result/isislsp.yaml

```diff
@@ -0,0 +1,380 @@
+      type: object
+      properties:
+        isis_router_name:
+          description: >-
+            The name of the ISIS Router.
+          type: string
+        lsps:
+          description: >-
+            One or more LSPs that are learned by this ISIS router.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.State'
+
+    IsisLsp.State:
+      description: >-
+        ISIS LSP.
+      type: object
+      properties:
+        lsp_id:
+          description: >-
+            LSP ID in the format, e.g. '640000000001-00-00'.
+            LSP ID consists of the System ID of a neighbor, the Pseudonode ID, and the LSP number.
+            The last two bytes represent Pseudonode ID and LSP number respectively.
+            A pseudonode is a logical representation of the LAN which is generated by a Designated Intermediate System (DIS)
+            on a LAN segment. If one LSP exceeds the maximum LSP size then it is sent in another LSP with the LSP number incremented by one.
+            A router's learned LSP gets refreshed by 'remaining_lifetime'. Then the sequence number is incremented by 1.
+          type: string
+        pdu_type:
+          description: >-
+            Link State PDU type.
+          type: string
+          enum: [level_1, level_2]
+        remaining_lifetime:
+          description: >-
+            Remaining lifetime in seconds before LSP expires.
+          type: integer
+          format: int64
+        sequence_number:
+          description: >-
+            Sequence number of the LSP.
+          type: integer
+          format: int64
+        pdu_length:
+          description: >-
+            Total length of the LSP.
+          type: integer
+        flags:
+          description: >-
+            LSP Type-Block flags.
+          $ref: '#/components/schemas/IsisLsp.Flags'
+        is_type:
+          description: |-
+            IS Type - bits 1 and 2 indicate the type of Intermediate System.
+            1 - ( i.e. bit 1 set) Level 1 Intermediate system.
+            2 - Unused value.
+            3 - (i.e. bits 1 and 2 set) Level 2 Intermediate system. 
+          type: integer
+        tlvs:
+          description: |-
+            It refers to Link State PDU State TLVs container.
+          $ref: '#/components/schemas/IsisLsp.Tlvs'
+
+    IsisLsp.Tlvs:
+      description: >-
+        This contains the list of TLVs present in one LSP.
+      type: object
+      properties:
+        hostname_tlvs:
+          description: >-
+            Array of Hostname TLVs ( type 137) present in this LSP.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.Hostname'
+        is_reachability_tlvs:
+          description: |-
+            Array of IS-Reachability TLVs (type 2) present in this LSP.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.IsReachability'
```

Comment:
Can we also rename all child object names, for consistency with the attribute name? IsReachability --> IsReachabilityTlv